### PR TITLE
[EK-25] Use `bosdyn_msgs` bundles

### PR DESCRIPTION
**Pull request by mhidalgo-bdai**
Depends on https://github.com/bdaiinstitute/bosdyn_msgs/pull/14. This patch updates this repository to work with the binary bundles that https://github.com/bdaiinstitute/bosdyn_msgs will release from now on. Changes are minimal (but changes nonetheless).

Tested manually by running examples against an Spot robot. I'm open to ideas on how to further test this.

### CODE REVIEW

**Comment by mhidalgo-bdai** [comment]
FTR CI will fail for now until a new `bosdyn_msgs` release is rolled out.

File: install_spot_ros2.sh
Comment (amessing-bdai) [inline install_spot_ros2.sh:1]: I think it would make sense to have a `VERSION="3.3.2"` and then use that through out as well. Likely will save us from a typo in the future.

Comment (mhidalgo-bdai) [reply install_spot_ros2.sh:1]: Good idea. Done in https://github.com/bdaiinstitute/spot_ros2/pull/234/commits/34a597492901413d8038cf173e59425bcee1a832.

**Comment by mhidalgo-bdai** [comment]
@amessing-bdai CI fails for the same reasons it failed over at `ros_utilities` and here for the `proto2ros_tests`: the dependency between container image and test job is not there. The test job runs on an older image.

We can re-work CI here like we did for `ros_utilities`. WDYT?

**Comment by mhidalgo-bdai** [comment]
This now needs https://github.com/bdaiinstitute/spot_wrapper/pull/90.

**Review by tcappellari-bdai — APPROVED** [review]
lgtm once the merge conflicts get resolved

**Review by amessing-bdai — APPROVED** [review]

**Comment by mhidalgo-bdai** [comment]
Local testing came out positive (thanks @khughes-bdai!). We are confident enough to move forward with this as-is.

**Review by khughes-bdai — APPROVED** [review]
LGTM!

**Pull request merged**
